### DISCUSSION TITLE
《中华人民共和国游行示威法》

### DISCUSSION THREAD

**Discussion by whyhhh** [WeNeedHome/SummaryOfLoanSuspension#453] [General]
https://suiping.gov.cn/web/front/news/detail.php?newsid=41607

根据第六条可以统一时间一起去市公安局提交申请

第四条公民在行使集会、游行、示威的权利的时候，必须遵守宪法和法律，不得反对宪法所确定的基本原则，不得损害国家的、社会的、集体的利益和其他公民的合法的自由和权利。
第五条集会、游行、示威应当和平地进行，不得携带武器、管制刀具和爆炸物，不得使用暴力或者煽动使用暴力。

第十三条集会、游行、示威的负责人对主管机关不许可的决定不服的，可以自接到决定通知之日起三日内，向同级人民政府申请复议，人民政府应当自接到申请复议书之日起三日内作出决定。

**Comment by whyhhh** [comment]
当然国家能给出解决办法就没必要到这一步了

**Reply by xiejun946** [reply]
越级上访都要被查水表，组织这个示威游行，我觉得肯定要被抓起来

**Reply by kennard520** [reply]
肯定要被抓起来的，这个法，只是用来给你看的

**Reply by kennard520** [reply]
如果上次被杀的不是安倍晋三 而是中国某个，中国人民会不会很高兴

**Reply by mccken** [reply]
> 如果上次被杀的不是安倍晋三 而是中国某个，中国人民会不会很高兴

得看在哪里发言，如果是国内社交平台，那我是愤怒的。
在推特和github上，在心底，我还是很高兴的

**Reply by yuanlimeisu** [reply]
荒谬，我tm游行还要向公安局申请，这他妈的谁能批准

**Reply by luuthink** [reply]
> 如果上次被杀的不是安倍晋三 而是中国某个，中国人民会不会很高兴

#可惜不是你#这个词条不就被封了

**Comment by zhengbiubiu** [comment]
放心，申请肯定不会批的，然后就是自己组织去游行就是非法集会，白衬衫伺候。

**Reply by whyhhh** [reply]
至少是最后的能够表达态度的办法呗

**Reply by kennard520** [reply]
成都一个楼盘维权的，都被特警抓走了，你损害了谁的利益

**Reply by whyhhh** [reply]
停贷是第一次施压，申请游行是第二次施压，会哭的孩子才有奶喝，天安门那是反政府了，肯定派武力的，停贷后啥结果都没有，又什么都不敢做，这也只能证明中产阶级的软弱性和妥协性

**Reply by whyhhh** [reply]
而且游行肯定不能马上游啊，除非第一次停贷全给你上征信了，不给活路

**Reply by xiejun946** [reply]
谈不上中产阶级，有个房子真谈不上，还是个烂尾楼

**Reply by Anchorj** [reply]
> 成都一个楼盘维权的，都被特警抓走了

请教：请问是何时的事情啊，这么凶

**Comment by xiejun946** [comment]
非法律专业的搞不了这个，还是法律工作者才能搞，申请法院行政复议才有可能

**Reply by CKwuliuqi** [reply]
在这里法律就是个P，法律的解释权在人家手里，人家想咋解释就咋解释，想搞这个先写个三年报告，再审批三年，再打回，有的是办法拿捏

**Reply by Sun-Cold** [reply]
你法我笑.jpg

**Reply by hclgdb** [reply]
”寻衅滋事“了解一下，嘿嘿。

**Comment by yuntun7zz** [comment]
这个申请肯定是通过不了的

**Comment by KisaraKira** [comment]
Tank Man WARNING
![图片](https://user-images.githubusercontent.com/108816528/179147039-2a6e7d8e-d000-42c0-86e0-e02a9eec3342.png)

**Reply by LvslGit** [reply]
这图也就在国外这些平台能放了。

**Reply by Novite5** [reply]
坦克也没把人怎么样。

**Reply by pjc752** [reply]
> 坦克也没把人怎么样。

坦克开出来想测试性能么？

**Reply by ghost** [reply]
> 坦克也没把人怎么样。

润滑一下履带，不要大惊小怪

**Reply by luuthink** [reply]
请回答1989 --- 梅花发卡

高考结束的那个夏天，她考上了北京的大学，而我报名参了军。

那天我同往常一样骑着自行车载上她，一路直到火车站，临别时,我亲手为她戴上了新买的梅花发卡。我记不清离别时被眼泪模糊的画面，只记得空气中她残留的发香，闻起来像梅花一样。

多年以后，我在部队清洗坦克履带时，在缝隙中发现了一支梅花发卡。

我记不清那天让我嗅觉失灵的腥味，只记得广场上那残留的血印，看起来像梅花一样。

**Comment by lettermessenger** [comment]
坦克准备就位，一群乌合之众，竟敢状告本官

**Comment by tencentbook** [comment]
感觉悬，国内有批准过正常示威游行的吗

**Reply by yanwuguangjun** [reply]
不知道有没有先例

**Reply by Sunny79928** [reply]
有过先例，当国家需要你上街的时候。例如99年南联盟大使馆事件，就是大学组织学生上街

**Comment by C-Joey** [comment]
那至少得等到二十大之后了
现在游行可能会违反防疫法 哈哈

**Comment by mehhss** [comment]
虽然不会启动坦克，但是赋红码还是很简单的

**Comment by toxin08x** [comment]
这个法可有用了，在法律层面上杜绝了所有老共不喜欢的游行。

**Comment by buyuanzuonuli** [comment]
敢游行直接抓起来伺候，我们这里上访都抓起来，何况示威呢。

**Comment by 505Games-oss** [comment]
如果真想信你就输了，法律是专门来限制咱们这群韭菜的。

**Comment by itsoursduty** [comment]
讲个笑话 上次佩洛西在台北落地，有人去美国大使馆门口抗议，都被晶哥逮起来了

**Comment by Dreaming-Pepper** [comment]
批不批，我觉得最好要去试试。
猜测肯定是不批的，要把过程录下来，全网直播，买水军搅浑舆论。
要做就要做到底，做一半就不做，一点阻碍就停下来，活该做韭菜。

**Comment by summerBreeze03** [comment]
“国家的一切权利属于人民”
